life/updated/02-life.adoc

```diff
@@ -1,189 +1,192 @@
+For lack of understanding, the enemy betrayed me and led me aside.
+By hearkening to him and the young people who were my neighbors,
+he persuaded me that the world had much that would be of great benefit to me,
+for I was young, and knew not what I might come to.
+I was left, by the death of my tender father, with hardly any friends,
 and in my distress and afflictions, willing to have a little rest and comfort,
 I lent an ear to the enemy of my soul, and let my mind go forth after fine clothes.
-When it was drawn out, it went without limit;
+When it was drawn out, it went without limit,
 and when I decked myself as finely and as choice as I could,
-it would hardly give me content; for when I had one new thing, and saw another,
+it would hardly give me content. When I had one new thing, and saw a second,
 or a third, I was as desirous of it as for the former, and so ever unsatisfied.
-The lying enemy had promised me rest and peace, but could not give it;
-a liar he is and ever was, my soul is at enmity with him;
-the Lord preserve me out of his snares, and my house also forever.
+The lying enemy had promised me rest and peace, but could not give it.
+A liar he is and ever was, and my soul is at enmity with him.
+May the Lord preserve me and my house out out of his snares forever.
 
-But though he had thus drawn out my mind, the Lord did not leave me;
-for many times I had a concern what would become of me;
-and if at any time I was drawn into mirth or laughter,
+But though he had thus drawn out my mind, the Lord did not leave me.
+For many times I had a concern what would become of me.
+And if at any time I was drawn into mirth or laughter,
 I would feel something smite my heart, and bring great heaviness over my spirit.
 I knew not what it was, and little thought it was the Lord,
 who was ever good and gracious, kind, merciful and slow to anger,
 and not willing people should run into destruction.
-
-I little thought he looked so narrowly to my ways;
-but since the Lord has been pleased to open my eyes,
-I can look back and admire his goodness; and blessed be his worthy name,
-and the right arm of his strength, who has early been my guide,
+I did not think the Lord looked so closely at my ways,
+but since He has been pleased to open my eyes,
+I can look back and admire his goodness. Blessed be His worthy name,
+and the right arm of His strength, who has early been my guide,
 and kept me in great degree from running into the evil of the world,
 which greatly attends young people.
-He took me by the hand, and led me when I knew it not, in my tender years;
-and if I had not hearkened unto the enemy, my condition had been well.
-As soon as he had drawn my mind into pride, and to take delight in fine clothes,
+He took me by the hand, and led me when I knew it not, in my tender years.
+And if I had not hearkened unto the enemy, my condition would have been well.
+For as soon as he had drawn my mind into pride, and to take delight in fine clothes,
```

Comment:
I think this would make more sense if you put it like this: And if I had not hearkened unto the enemy, my condition would have been well**, but** as soon as he had drawn my mind into pride, and to take delight in fine clothes, they became my burden.